data_subscriptions/notifications/user_notification_dispatcher.py

```diff
@@ -72,20 +81,21 @@ def newDatasetMsgFilter(self, activitesList):
     def datasetUpdateMsgFilter(self, activitesList):
         return activitesList.get("packages", [])
 
-    @property
+    @ property
```

Comment:
@luccasmmg is there a reason you're putting whitespace here?

Reply:
I think that was my linter, went back to previous version